README.md

```diff
@@ -218,20 +220,28 @@ The `CC_TEST_REPORTER_ID` environment variable must be configured.
 
 ## Development
 
+### shellcheck
+
+To run the [shellcheck](https://github.com/koalaman/shellcheck), run
+
+```sh
+docker compose run --rm shellcheck hooks/*
```

Comment:
Doesn't really matter that much, but `docker compose` is inconsistent with `docker-compose` used below in this readme. Maybe we should update the ones below to `docker compose` without the hyphen (I think that's the "new" way of doing it, without the hyphen?).